SwiftLint.xcodeproj/project.pbxproj

```diff
@@ -1216,6 +1220,20 @@
 			shellPath = /bin/sh;
 			shellScript = "if which swiftlint >/dev/null; then\n  swiftlint\nelse\n  echo \"SwiftLint does not exist, download from https://github.com/realm/SwiftLint\"\nfi";
 		};
+		E86C76171EE0CC1B0081860F /* Run Sourcery */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			name = "Run Sourcery";
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "if which sourcery >/dev/null; then\n  make sourcery\nelse\n  echo \"Sourcery not found, install with 'brew install sourcery'\"\nfi";
```

Comment:
1. Travis doesn't appear to have Sourcery installed.
2. It shouldn't be an error (or a warning even) if you don't have Sourcery installed but you're building SwiftLint via Xcode. It's only really useful for contributors.